packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/utils/forms.js

```diff
@@ -505,6 +514,21 @@ const forms = {
         } else if (type === 'uid') {
           const uidItems = [
             [{ ...fields.default, disabled: Boolean(data.targetField), type: 'text' }],
+            [
+              {
+                autoFocus: false,
+                label: {
+                  id: getTrad('form.attribute.item.text.regex'),
+                },
+                name: 'regex',
+                disabled: Boolean(data.targetField),
+                type: 'text',
+                validations: {},
+                description: {
```

Comment:
I am not sure the provided description is correct in this case the feature adds a regex to the uid field and does not make sure that the content of the field matches the regex. What do you think about adding a clearer description so there is no misunderstanding between the validation regex which is applied for the text field and the feature which generates the uid with a random regex?

Reply:
Just like `text` field,  the new regex setting also validate the content of the field.
I agree we could append a description about generating the random uid, but I will need help for the translations.